services/js/src/main/scala/org/apache/gearpump/dashboard/DashboardApp.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.gearpump.dashboard
+
+import com.greencatsoft.angularjs.Angular
+import org.apache.gearpump.dashboard.services._
+
+import scala.scalajs.js.JSApp
+import scala.scalajs.js.annotation.JSExport
+
+@JSExport
+object DashboardApp extends JSApp {
```

Comment:
Frankly speaking, scala binding for angular is not that promsing. Until recently(1 day ago), it has a big change of the HttpService interface. 
https://github.com/greencatsoft/scalajs-angular/pull/50/files

Depending on this changing library is riskly. 

I can see your argument that restapi.js depends on javascript version of angular. Then  in that case, RestApiService.scala should not be incldued until the discussion is clear. 

Maybe for now the scalajs part should only expose the serializer and deserializer which is implemented with upickle.